### PR TITLE
Make excludes apps list setting up by config file

### PR DESCRIPTION
**Pull request by CPatchane**
Make excludes apps list setting up by a config file, will allow to customise this list

### CODE REVIEW

**Comment by CPatchane** [comment]
[(𝐒) [BAR] ETUO j'ai accès à Settings via la liste des applications](https://trello.com/c/gmAAk8wN/1060-%F0%9D%90%92-bar-eto-jai-acc%C3%A8s-%C3%A0-settings-via-la-liste-des-applications)

File: src/config/excludedApps.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+- settings
```
Comment (nono) [inline src/config/excludedApps.yaml]: Why a YAML and not a JSON file?

**Comment by gregorylegarec** [comment]
I guess we can just override this configuration file to customize or Cozy-Bar ? If so, LGTM.

Same question as @nono by the way. Why YAML ?

File: src/config/excludedApps.yaml (hunk 1 above)
Comment (CPatchane) [reply src/config/excludedApps.yaml]: We were already using those files for configs in cozy-bar so, it allows to add comments in configuration even if for here YAML or JSON doesn't matter

**Pull request merged**